core/src/main/resources/jenkins/model/Messages_de.properties

```diff
@@ -25,40 +25,43 @@ Hudson.Computer.Caption=Master
 Hudson.Computer.DisplayName=master
 Hudson.ControlCodeNotAllowed=Kontrollcodes nicht erlaubt: {0}
 Hudson.DisplayName=Jenkins
-Hudson.JobAlreadyExists=Es existiert bereits ein Job ''{0}''
-Hudson.NoJavaInPath=java ist nicht in Ihrem PATH-Suchpfad. Eventuell sollten Sie <a href=''{0}/configure''>JDKs konfigurieren</a>.
+Hudson.JobAlreadyExists=Es existiert bereits ein Job \u201E{0}\u201C
+Hudson.NoJavaInPath=java ist nicht in Ihrem PATH-Suchpfad. Eventuell sollten Sie <a href="{0}/configure">JDKs konfigurieren</a>.
 Hudson.NoName=Kein Name angegeben
 Hudson.NodeBeingRemoved=Knoten wird entfernt
-Hudson.UnsafeChar=''{0}'' ist kein ''sicheres'' Zeichen
-Hudson.JobNameConventionNotApplyed=Der Jobname ''{0}'' folgt nicht der Namenskonvention ''{1}''
-Hudson.ViewAlreadyExists=Es existiert bereits eine Ansicht mit dem Namen "{0}".
+Hudson.UnsafeChar=\u201E{0}\u201C ist kein \u201Esicheres\u201C Zeichen
+Hudson.JobNameConventionNotApplyed=Der Elementname \u201E{0}\u201C folgt nicht der Namenskonvention \u201E{1}\u201C
+Hudson.ViewAlreadyExists=Es existiert bereits eine Ansicht mit dem Namen \u201E{0}\u201C.
 Hudson.ViewName=Alle
 Hudson.NotUsesUTF8ToDecodeURL=\
   Ihr Container verwendet kein UTF-8, um URLs zu dekodieren. Falls Sie Nicht-ASCII-Zeichen \
```

Comment:
Yes, this has bothered me a bit as well.

[Wikipedia calls it *web container*](https://en.wikipedia.org/wiki/Web_container), perhaps we should use that? *Web-Container*?